src/main/java/io/opentelemetry/semconv/SemanticAttributes.java

```diff
@@ -329,263 +345,565 @@ public final class SemanticAttributes {
   /** The basename of the file. */
   public static final AttributeKey<String> LOG_FILE_NAME = stringKey("log.file.name");
 
-  /** The full path to the file. */
-  public static final AttributeKey<String> LOG_FILE_PATH = stringKey("log.file.path");
-
   /** The basename of the file, with symlinks resolved. */
   public static final AttributeKey<String> LOG_FILE_NAME_RESOLVED =
       stringKey("log.file.name_resolved");
 
+  /** The full path to the file. */
+  public static final AttributeKey<String> LOG_FILE_PATH = stringKey("log.file.path");
+
   /** The full path to the file, with symlinks resolved. */
   public static final AttributeKey<String> LOG_FILE_PATH_RESOLVED =
       stringKey("log.file.path_resolved");
 
-  /** The type of memory. */
-  public static final AttributeKey<String> TYPE = stringKey("type");
+  /**
+   * The name of the connection pool; unique within the instrumented application. In case the
+   * connection pool implementation does not provide a name, then the <a
+   * href="/docs/database/database-spans.md#connection-level-attributes">db.connection_string</a>
+   * should be used
+   */
+  public static final AttributeKey<String> POOL_NAME = stringKey("pool.name");
+
+  /** The state of a connection in the pool */
+  public static final AttributeKey<String> STATE = stringKey("state");
 
   /**
-   * Name of the memory pool.
+   * Name of the buffer pool.
    *
    * <p>Notes:
    *
    * <ul>
    *   <li>Pool names are generally obtained via <a
-   *       href="https://docs.oracle.com/en/java/javase/11/docs/api/java.management/java/lang/management/MemoryPoolMXBean.html#getName()">MemoryPoolMXBean#getName()</a>.
+   *       href="https://docs.oracle.com/en/java/javase/11/docs/api/java.management/java/lang/management/BufferPoolMXBean.html#getName()">BufferPoolMXBean#getName()</a>.
    * </ul>
    */
-  public static final AttributeKey<String> POOL = stringKey("pool");
```

Comment:
Until we resolve #6 we need to do the painful, error prone process of updating the [SemanticAttributes.javaj2](https://github.com/open-telemetry/semantic-conventions-java/blob/main/buildscripts/templates/SemanticAttributes.java.j2) template manually with deprecation notices for all the deleted attributes. This will be particularly painful for the 1.22.0 upgrade 🙁